README.md

```diff
@@ -90,10 +90,72 @@ To start the local webserver and have gulp watch for changes:
 npm start
 ```
 
-#### Descriptions for optional `gulp [ flags ]` task prefixes
+### Descriptions for optional `gulp [ flags ]` task prefixes
 
 These flags are 100% optional and can be omitted from any tasks that are
 affected by them.
 
 - `no-test` This flag disables linters and tests for assets
 - `production` This flag enables minification and compression of assets
+
+## Deployment
+
+The microsite is deployed on [cloud.gov] [cg-homepage]. To read the `cloud.gov`
+documentation, [click here] [cg-docs]. The following documentation assumes you
+have the `cf` binary installed on your machine. [Please read _Setting up the
+command line_] [cg-docs-cli-install].
+
+[cg-homepage]: https://cloud.gov "Cloud.gov: Homepage"
+[cg-docs]: https://docs.cloud.gov "Cloud.gov: Documentation"
+[cg-docs-cli-install]: https://docs.cloud.gov/getting-started/setup/ "Cloud.gov: Setting up the command line"
+
+### Automated deployment
+
+This project uses [Travis-CI] [tci-homepage] for continuous deployment. Our
+current process deploys our `staging` branch and our `master` branch to their
+own [`staging`] [ffd-staging] and [`production`] [ffd-production] URLs.
+
+[tci-homepage]: https://travis-ci.org "Travis-CI: Homepage"
+[ffd-staging]: https://ffd-microsite-staging.apps.cloud.gov "Federal Front Door: Staging"
+[ffd-production]: https://labs.usa.gov "Federal Front Door: Production"
+
+All new Pull Requests must target this repository's `staging` branch. You can
+use [this link to ensure the base branch is set correctly] [gh-base-branch].
+
+[gh-base-branch]: https://github.com/18F/ffd-microsite/compare/staging...staging "Github: Submit a new Pull Request"
+
+### Manual deployment
+
+Using the `cf` command-line tool, you can run a manual deployment to either
+`staging` or `production` by targeting the corresponding organization / space
+and as long as you have access to `cf push` the target. More information on
+deploying to `cloud.gov` can be found [here] [cg-deploy-hw] and [here] [cg-deploy-ss].
+
+[cg-deploy-hw]: https://docs.cloud.gov/getting-started/your-first-deploy/ "Cloud.gov: Your First Deploy"
+[cg-deploy-ss]: https://docs.cloud.gov/apps/static/ "Cloud.gov: Deploying Static Sites"
+
+> Manual deployments aren't necessary as all updates should go through Travis-CI.
```

Comment:
Should this be more prominent? I wasn't sure if the `Manual deployment` section is necessary if we're going to stick to `Automated deployment`. but this information is valuable in case `Travis-CI` is down. Thoughts @juliaelman @maya @yozlet? Let me know if we want to add anymore to this section as well. :bow: